packages/pages/src/util/processEnvVariables.test.ts

```diff
@@ -4,30 +4,34 @@ import { processEnvVariables } from "./processEnvVariables.js";
 describe("processEnvVariables", () => {
   beforeAll(() => {
     global.process.env = {
-      NODE_ENV: "development",
+      NODE_ENV: "production",
       VITE_KEY: "pk.abcdefghij",
       YEXT_PUBLIC_KEY: "pk.0123456789",
+      SECRET: "secret",
     };
   });
 
-  it("filters .env for Vite specific variables", () => {
+  it("filters .env for Vite specific variables for production", () => {
     const env = processEnvVariables();
-
     expect(Object.keys(env).length).toEqual(1);
     expect(env.VITE_KEY).toEqual(`"pk.abcdefghij"`);
   });
 
-  it("filters .env for Vite specific variables with custom prefix", () => {
+  it("filters .env for Vite specific variables with custom prefix for production", () => {
     const env = processEnvVariables("YEXT_PUBLIC");
 
     expect(Object.keys(env).length).toEqual(1);
     expect(env.YEXT_PUBLIC_KEY).toEqual(`"pk.0123456789"`);
   });
 
-  it("does not stringify values", () => {
-    const env = processEnvVariables("YEXT_PUBLIC", false);
+  it("returns all env vars for development", () => {
+    global.process.env.NODE_ENV = "development";
+    const env = processEnvVariables("YEXT_PUBLIC");
```

Comment:
We could have an example with another key, like VITE

Reply:
The key actually doesn't matter in this case as it returns everything. Ack.